src/main/java/it/gov/pagopa/payhub/activities/service/paymentsreporting/PaymentsReportingMapperService.java

```diff
@@ -0,0 +1,65 @@
+	 * @param ingestionFlowFileDTO the ingestion metadata containing information about the processing flow.
+	 * @return a list of `PaymentsReportingDTO` objects, one for each individual payment in the flow.
+	 */
+	public List<PaymentsReportingDTO> mapToDtoList(CtFlussoRiversamento ctFlussoRiversamento, IngestionFlowFileDTO ingestionFlowFileDTO) {
+
+		PaymentsReportingDTO.PaymentsReportingDTOBuilder builder = PaymentsReportingDTO.builder()
+			.creationDate(Instant.ofEpochMilli(System.currentTimeMillis()))
+			.lastUpdateDate(Instant.ofEpochMilli(System.currentTimeMillis()))
+			.acquiringDate(new Date(ingestionFlowFileDTO.getCreationDate().getTime()).toLocalDate())
+			.organizationId(ingestionFlowFileDTO.getOrg().getOrgId())
+			.ingestionFlowFileId(ingestionFlowFileDTO.getIngestionFlowFileId())
+			.pspIdentifier(ctFlussoRiversamento.getIstitutoMittente().getDenominazioneMittente())
+			.flowIdentifierCode(ctFlussoRiversamento.getIdentificativoFlusso())
+			.flowDateTime(ctFlussoRiversamento.getDataOraFlusso().toGregorianCalendar().toZonedDateTime().toLocalDate())
+			.regulationUniqueIdentifier(ctFlussoRiversamento.getIdentificativoUnivocoRegolamento())
+			.regulationDate(ctFlussoRiversamento.getDataRegolamento().toGregorianCalendar().toZonedDateTime().toLocalDate())
+			.senderPspName(ctFlussoRiversamento.getIstitutoMittente().getDenominazioneMittente())
+			.senderPspCode(ctFlussoRiversamento.getIstitutoMittente().getIdentificativoUnivocoMittente().getCodiceIdentificativoUnivoco())
+			.senderPspType(ctFlussoRiversamento.getIstitutoMittente().getIdentificativoUnivocoMittente().getTipoIdentificativoUnivoco().value())
+			.receiverOrganizationName(ctFlussoRiversamento.getIstitutoRicevente().getDenominazioneRicevente())
+			.receiverOrganizationCode(ctFlussoRiversamento.getIstitutoRicevente().getIdentificativoUnivocoRicevente().getCodiceIdentificativoUnivoco())
+			.receiverOrganizationType(ctFlussoRiversamento.getIstitutoRicevente().getIdentificativoUnivocoRicevente().getTipoIdentificativoUnivoco().value())
+			.totalPayments(ctFlussoRiversamento.getDatiSingoliPagamenti().size())
+			.totalAmountCents(ctFlussoRiversamento.getImportoTotalePagamenti().movePointRight(2).longValueExact())
+			.totalPayments(ctFlussoRiversamento.getNumeroTotalePagamenti().longValueExact())
+			.bicCodePouringBank(ctFlussoRiversamento.getCodiceBicBancaDiRiversamento());
+
+		return ctFlussoRiversamento.getDatiSingoliPagamenti().stream()
+			.map(item -> builder
+				.creditorReferenceId(item.getIdentificativoUnivocoVersamento())
+				.regulationId(item.getIdentificativoUnivocoRiscossione())
+				.transferIndex(item.getIndiceDatiSingoloPagamento())
+				.amountPaidCents(item.getSingoloImportoPagato().movePointRight(2).longValueExact())
+				.paymentOutcomeCode(item.getCodiceEsitoSingoloPagamento())
+				.payDate(item.getDataEsitoSingoloPagamento().toGregorianCalendar().toZonedDateTime().toLocalDate())
```

Comment:
check if the field contains just date or date/time

Reply:
it is just a date